src/pages/auth/login/index.tsx

```diff
@@ -24,23 +28,36 @@ import { LoginFileds, initialValues } from './const'
 
 export const LoginComponent = () => {
   const dispatch = useAppDispatch()
+  const { executeRecaptcha } = useGoogleReCaptcha()
   const { isLoading } = useAppSelector(userInfoSelector)
   const { id, isUse2FA } = useAppSelector(userDataSelector)
 
-  const onSuccessCallback = (data: AuthResponse) => dispatch(saveUserDataThunk(data))
-  const onErrorCallback = (err: string) => toast.error(err)
-  const handleCloseModal = () => dispatch(cancelUser2faAction())
-
-  const setAttemptsLeftCount = (count: number) => dispatch(setAttemptsLeftCountAction(count))
-
   const formik = useFormik({
     initialValues,
     validationSchema: loginValidationSchema,
     validateOnBlur: true,
     validateOnChange: true,
-    onSubmit: (data) => dispatch(loginThunk({ ...data, setAttemptsLeftCount }))
+    onSubmit: noopFunction
```

Comment:
Why you use `noop` here?

Reply:
Because onSubmit will not call, and its like a plug